### ISSUE TITLE
Subtidal vs intertidal acreage (DF)

### ISSUE THREAD

**Issue by mmosesOCMP** [orcoastalmgmt/orSeaSketch#26]
On site reports and attributes: It’s really important to distinguish subtidal and intertidal area.  Perhaps the acreages of each can be listed in the Geography section.

**Comment by tchaddad** [comment]
So the issue with this item is that we do not have a line for the lower end of the intertidal. 

We've published a -5m line, do you want us to use that line?

**Comment by mmosesOCMP** [comment]
@tchaddad Yes, that is the subtidal limit of intertidal habitat defined in the Strategy.

**Comment by mmosesOCMP** [comment]
The present issue with this is that Intertidal and Subtidal Substrate reports are querying two different datasets and they do not complement one another. Intertidal Substrates is querying the CMECS data, but is not distinguishing habitat types (inter vs subtidal). It should be two queries which categorize anything below the -5m line as deep, and anything between -5m and some other line as shallow. This can be accomplished by clipping the Surface Lithology layer to the extent of the CMECS layer.

**Comment by mmosesOCMP** [comment]
Clarification & conclusion: 

- clip surface lithology layer to the extent of the CMECS intertidal layer
- rename two queries: 'Shallow Substrates' and 'Deeper Substrates'

**Comment by mmosesOCMP** [comment]
![image](https://user-images.githubusercontent.com/62967932/101220365-67a80600-363a-11eb-9d70-cddfd64e0f2f.png)
